bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -151,9 +153,10 @@
      */
     public static final ChannelType SYSTEM_MOTION = ChannelTypeBuilder
             .state(new ChannelTypeUID(BINDING_ID, "motion"), "Motion", "Switch")
-            .withDescription("Motion detected by the device").withCategory("Motion").withStateDescription(
+            .withDescription("Motion detected by the device").withCategory("Motion")
+            .withStateDescription(
                     StateDescriptionFragmentBuilder.create().withReadOnly(true).build().toStateDescription())
-            .build();
+            .withTags(Arrays.asList(SystemTags.CAP_SWITCHABLE)).build();
```

Comment:
As this is read only, I am against "Capability:Switchable" but for "Capability:Measurement". I addition we need "Property:Presence" here. And what about "Purpose:Presence"?

Reply:
I agree with "Capability:Measurement" and "Property:Presence". The **purpose** however should be left to the user, because it depends on his scenario where he is going to use the device for. At least that's my opinion about it.

Reply:
ok